elide-datastore/elide-datastore-jpa/src/test/java/com/yahoo/elide/datastores/jpa/JpaDataStoreSupplier.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.jpa;
+
+import com.yahoo.elide.core.DataStore;
+import com.yahoo.elide.datastores.jpa.transaction.NonJtaTransaction;
+import com.yahoo.elide.utils.ClassScanner;
+
+import example.Parent;
+
+import org.hibernate.MappingException;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.boot.spi.MetadataImplementor;
+import org.hibernate.cfg.Environment;
+import org.hibernate.jpa.AvailableSettings;
+import org.hibernate.tool.hbm2ddl.SchemaExport;
+import org.hibernate.tool.schema.TargetType;
+
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+import javax.persistence.Entity;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+/**
+ * Supplier of Hibernate 5 Data Store.
+ */
+public class JpaDataStoreSupplier implements Supplier<DataStore> {
+    private static final String JDBC = "jdbc:h2:mem:root;IGNORECASE=TRUE";
+    private static final String ROOT = "root";
+
+    @Override
+    public DataStore get() {
+        Map<String, Object> options = new HashMap<>();
+        ArrayList<Class<?>> bindClasses = new ArrayList<>();
+
+        try {
+            bindClasses.addAll(ClassScanner.getAnnotatedClasses(Parent.class.getPackage(), Entity.class));
```

Comment:
I think you'll need more than this now.  This only covers one package.  You will need to include elide-example-models as well.